tests/shutdown/cleanup_before_shutdown.pm

```diff
@@ -48,7 +48,7 @@ END_SCRIPT
     # Configure serial consoles for virtio support
     # poo#18860 Enable console on hvc0 on SLES < 12-SP2
     # poo#44699 Enable console on hvc1 to fix login issues on ppc64le
-    if (!check_var('VIRTIO_CONSOLE', 0)) {
+    if (check_var('VIRTIO_CONSOLE', 1)) {
```

Comment:
@asmorodskyi IMHO this is wrong. If not defined, it should be treated as 1. But not sure if it's in this stage as 1 propagated into vars. Our discussion about it related in the past: https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/6073

Reply:
ok , I will revert this